### ISSUE TITLE
加载ernie预训练模型在inews情感分析数据集上得到的准确率是0.103是怎么呢?(train epoch =100)

### ISSUE THREAD

**Issue by caixxiong** [ChineseGLUE/ChineseGLUE#91]
I0721 19:18:55.035024 139930260309824 run_classifier.py:1507]   eval_accuracy = 0.1031031
INFO:tensorflow:  eval_loss = 9.384537
I0721 19:18:55.035102 139930260309824 run_classifier.py:1507]   eval_loss = 9.384537
INFO:tensorflow:  global_step = 16734
I0721 19:18:55.035161 139930260309824 run_classifier.py:1507]   global_step = 16734
INFO:tensorflow:  loss = 9.385627
I0721 19:18:55.035216 139930260309824 run_classifier.py:1507]   loss = 9.385627

**Comment by caixxiong** [comment]
刚才去看了代码, 发现代码里边在eval test时, 手动将标签全部置为0了, 所以accuracy相当于是所有0类数/总数=99/1000, 约为0.1😂😂
